File: php-sdk/src/triplink/vcc/card/client/operate/OperateResponse.php
<?php
namespace triplink\vcc\card\client\operate;
class OperateResponse {

    var $resultstatus;

    var $resultRespCode;

    var $resultRespMsg;

    var $channelType;

    var $ccOpId;

    var $requestId;

    var $requestTime;

    var $requestType;

    var $subRequestType;

    var $merchantName;

    var $cardInfo;

    var $respExtra;
    /**
     * @return the $resultstatus
     */
    public function getResultstatus()
    {
        return $this->resultstatus;
    }

    /**
     * @return the $resultRespCode
     */
    public function getResultRespCode()
    {
        return $this->resultRespCode;
    }

    /**
     * @return the $resultRespMsg
     */
    public function getResultRespMsg()
    {
        return $this->resultRespMsg;
    }

    /**
     * @return the $channelType
     */
    public function getChannelType()
    {
        return $this->channelType;
    }

    /**
     * @return the $ccOpId
     */
    public function getCcOpId()
    {
        return $this->ccOpId;
    }

    /**
     * @return the $requestId
     */
    public function getRequestId()
    {
        return $this->requestId;
    }

    /**
     * @return the $requestTime
     */
    public function getRequestTime()
    {
        return $this->requestTime;
    }

    /**
     * @return the $requestType
     */
    public function getRequestType()
    {
        return $this->requestType;
    }

    /**
     * @return the $subRequestType
     */
    public function getSubRequestType()
    {
        return $this->subRequestType;
    }

    /**
     * @return the $merchantName
     */
    public function getMerchantName()
    {
        return $this->merchantName;
    }

    /**
     * @return the $cardInfo
     */
    public function getCardInfo()
    {
        return $this->cardInfo;
    }

    /**
     * @return the $respExtra
     */
    public function getRespExtra()
    {
        return $this->respExtra;
    }

    /**
     * @param field_type $resultstatus
     */
    public function setResultstatus($resultstatus)
    {
        $this->resultstatus = $resultstatus;
    }

    /**
     * @param field_type $resultRespCode
     */
    public function setResultRespCode($resultRespCode)
    {
        $this->resultRespCode = $resultRespCode;
    }

    /**
     * @param field_type $resultRespMsg
     */
    public function setResultRespMsg($resultRespMsg)
    {
        $this->resultRespMsg = $resultRespMsg;
    }

    /**
     * @param field_type $channelType
     */
    public function setChannelType($channelType)
    {
        $this->channelType = $channelType;
    }

    /**
     * @param field_type $ccOpId
     */
    public function setCcOpId($ccOpId)
    {
        $this->ccOpId = $ccOpId;
    }

    /**
     * @param field_type $requestId
     */
    public function setRequestId($requestId)
    {
        $this->requestId = $requestId;
    }

    /**
     * @param field_type $requestTime
     */
    public function setRequestTime($requestTime)
    {
        $this->requestTime = $requestTime;
    }

    /**
     * @param field_type $requestType
     */
    public function setRequestType($requestType)
    {
        $this->requestType = $requestType;
    }

    /**
     * @param field_type $subRequestType
     */
    public function setSubRequestType($subRequestType)
    {
        $this->subRequestType = $subRequestType;
    }

    /**
     * @param field_type $merchantName
     */
    public function setMerchantName($merchantName)
    {
        $this->merchantName = $merchantName;
    }

    /**
     * @param field_type $cardInfo
     */
    public function setCardInfo($cardInfo)
    {
        $this->cardInfo = $cardInfo;
    }

    /**
     * @param field_type $respExtra
     */
    public function setRespExtra($respExtra)
    {
        $this->respExtra = $respExtra;
    }

    
    
}
?>

File: src/main/java/com/ctrip/ccard/creditcard/vcc/bean/OperateResponse.java
package com.ctrip.ccard.creditcard.vcc.bean;

/**
 * Description:
 */
public class OperateResponse {
    /**
     * 结果状态
     * PROCESS：处理中
     * SUCCEED：成功
     * FAILURE：失败
     * EXCEPTION：异常
     */
    private String resultstatus;
    /**
     * 结果码
     */
    private String resultRespCode;
    /**
     * 结果码描述
     */
    private String resultRespMsg;
    /**
     * 发卡通道类型
     */
    private String channelType;
    /**
     * 本系统生成的唯一流水号
     */
    private String ccOpId;
    /**
     * 请求流水号
     */
    private String requestId;
    /**
     * 请求时间
     */
    private String requestTime;
    /**
     * 请求类型
     */
    private String requestType;
    /**
     * 请求子类型
     */
    private String subRequestType;
    /**
     * 请求商户名
     */
    private String merchantName;
    /**
     * 卡信息请求
     */
    private CardInfo cardInfo;
    /**
     * 响应扩展数据
     */
    private String respExtra;

    public String getResultstatus() {
        return resultstatus;
    }

    public void setResultstatus(String resultstatus) {
        this.resultstatus = resultstatus;
    }

    public String getResultRespCode() {
        return resultRespCode;
    }

    public void setResultRespCode(String resultRespCode) {
        this.resultRespCode = resultRespCode;
    }

    public String getResultRespMsg() {
        return resultRespMsg;
    }

    public void setResultRespMsg(String resultRespMsg) {
        this.resultRespMsg = resultRespMsg;
    }

    public String getChannelType() {
        return channelType;
    }

    public void setChannelType(String channelType) {
        this.channelType = channelType;
    }

    public String getCcOpId() {
        return ccOpId;
    }

    public void setCcOpId(String ccOpId) {
        this.ccOpId = ccOpId;
    }

    public String getRequestId() {
        return requestId;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

    public String getRequestTime() {
        return requestTime;
    }

    public void setRequestTime(String requestTime) {
        this.requestTime = requestTime;
    }

    public String getRequestType() {
        return requestType;
    }

    public void setRequestType(String requestType) {
        this.requestType = requestType;
    }

    public String getSubRequestType() {
        return subRequestType;
    }

    public void setSubRequestType(String subRequestType) {
        this.subRequestType = subRequestType;
    }

    public String getMerchantName() {
        return merchantName;
    }

    public void setMerchantName(String merchantName) {
        this.merchantName = merchantName;
    }

    public CardInfo getCardInfo() {
        return cardInfo;
    }

    public void setCardInfo(CardInfo cardInfo) {
        this.cardInfo = cardInfo;
    }

    public String getRespExtra() {
        return respExtra;
    }

    public void setRespExtra(String respExtra) {
        this.respExtra = respExtra;
    }

    @Override
    public String toString() {
        return "OperateResponse{" +
                "resultstatus='" + resultstatus + '\'' +
                ", resultRespCode='" + resultRespCode + '\'' +
                ", resultRespMsg='" + resultRespMsg + '\'' +
                ", channelType='" + channelType + '\'' +
                ", ccOpId='" + ccOpId + '\'' +
                ", requestId='" + requestId + '\'' +
                ", requestTime='" + requestTime + '\'' +
                ", requestType='" + requestType + '\'' +
                ", subRequestType='" + subRequestType + '\'' +
                ", merchantName='" + merchantName + '\'' +
                ", cardInfo=" + cardInfo +
                ", respExtra='" + respExtra + '\'' +
                '}';
    }
}
